packages/datadog-metrics-tracker/src/base.ts

```diff
@@ -18,6 +18,17 @@ export abstract class MetricsTracker {
     this.sendPointsToDatadog = this.sendPointsToDatadog.bind(this);
   }
 
+  protected async trackPoints(
+    measurementName: string,
+    tags: { [name: string]: string },
+    values: { [name: string]: number },
+  ) {
+    for (let name in values) {
```

Comment:
Can you do 
`for (const [name, value] in Object.entries(values)`
?

Reply:
will do